buildenvs/base.Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2022, NEC Europe Ltd., Unikraft GmbH, and The KraftKit Authors.
 # Licensed under the BSD-3-Clause License (the "License").
 # You may not use this file except in compliance with the License.
-ARG DEBIAN_VERSION=bookworm-20240513
+ARG DEBIAN_VERSION=bookworm-20241223
```

Comment:
If you do this, then do it for them all

![image](https://github.com/user-attachments/assets/6a2d204d-09e3-44bd-89b7-f1b999203cfd)

Reply:
good point